test/studies/bale/histogram/histo-atomics-forall-opt.comm-gasnet.good

```diff
@@ -0,0 +1 @@
+185 194 208 213 206 205 190 202 169 200 161 195 191 187 190 199 234 191 218 187 228 200 214 213 229 199 192 183 184 200 192 218 206 199 203 197 185 191 203 205 229 184 185 216 197 195 197 200 211 215 229 177 213 189 184 199 197 235 213 180 207 213 202 204 169 190 206 186 185 196 194 197 213 206 201 209 210 189 211 201
```

Comment:
Why is nothing being reported here, but we are reporting for comm-none?

Reply:
Because gasnet atomics use on statements which make the optimization not fire.